gateway/modules/crud/dataloader.go

```diff
@@ -173,7 +174,9 @@ func (graph *Module) dataLoaderBatchFn(c context.Context, keys dataloader.Keys)
 	req := model.ReadRequest{Find: map[string]interface{}{"$or": holder.getWhereClauses()}, Operation: utils.All, Options: &model.ReadOptions{}}
 
 	// Fire the merged request
-	res, err := graph.crud.Read(ctx, dbType, graph.project, col, &req)
+
+	req.Options.Prefix = ""
```

Comment:
Add a comment on why this line is so important. Also put a note to not remove this